### ISSUE TITLE
Zookeeper is spaming with "old client" messages

### ISSUE THREAD

**Issue by makitka2007** [andreas-schroeder/kafka-health-check#33]
When using kafka-health-check, there are a lot of such warning messages every few seconds in Zookeeper (3.4.12 version)
```Connection request from old client /<ip>:<port>; will be dropped if server is in r-o mode```

Internet says that this warning relates to C client library which doesn't support read only mode comparing to Java one.

When i stop health-checker, these spam warnings disappear.
